pkg/collector/corechecks/net/ntp.go

```diff
@@ -78,7 +79,19 @@ func (c *ntpConfig) parse(data []byte, initData []byte) error {
 	}
 
 	c.instance = instance
-	if c.instance.Host != "" {
+	var localNtpServers []string
+	var err error
+	if c.instance.UseLocalDefinedServers {
+		localNtpServers, err = getLocalServers()
+		if err != nil {
+			return err
+		}
+		log.Infof("Use local defined servers: %v", localNtpServers)
+	}
+
+	if len(localNtpServers) > 0 {
+		c.instance.Hosts = localNtpServers
```

Comment:
The behavior is inconsistent here between windows and linux. If you find 0 ntp server on linux `parse` will error out (see my other comment). On windows you will get default servers. Is this intended ?

Reply:
Fixed.